data_generation/expand.py

```diff
@@ -27,28 +27,51 @@ def get_args():
     )
     # if file is moved in another directory level relative to the root (currently in root/data_generation), this needs to be changed
     root_directory = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-    parser.add_argument("-d", "--data_path", default=root_directory + "/Data/train")
-    parser.add_argument("-s", "--save_dir", default=root_directory + "/Data/expand")
+    parser.add_argument("-d", "--data_path", default=root_directory + "/Data")
+    parser.add_argument("--dataset", choices=['train', 'val', 'test'], default='train')
+    parser.add_argument("-s", "--save_dir", default=root_directory + "/Data")
     args = parser.parse_args()
```

Comment:
Do you have no control over the amount of graphs that are created from every graph?

Reply:
added it myself